Client/src/Components/CreateMaintenanceWindow/index.jsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Stack, Typography } from "@mui/material";
+import "./index.css";
+import React, { useState } from "react";
+import Button from "../Button";
+import Back from "../../assets/icons/left-arrow.svg?react";
+import Select from "../Inputs/Select";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import dayjs from "dayjs";
+import { DatePicker } from "@mui/x-date-pickers/DatePicker";
+import { MobileTimePicker } from "@mui/x-date-pickers/MobileTimePicker";
+
+const directory = {
+  title: "Create a maintenance window",
+  description: "Your pings won’t be sent in this time frame.",
+};
+
+const repeatOptions = [
+  { name: "Don't repeat", _id: 1 },
+  { name: "Repeat daily", _id: 2 },
+  { name: "Repeat weekly", _id: 3 },
```

Comment:
Convention is genreally that the unique identifier should be the first key in an object if it is a map type object.

Unfamiliar users will be confused by this and may asusme that `name` is the unique identifier rathern than `_id`

```
{_id: 1, name: "Don't repeat}"
```
is clearer and doesn't leave room for error